go.sum

```diff
@@ -379,7 +379,6 @@ golang.org/x/crypto v0.0.0-20190426145343-a29dc8fdc734/go.mod h1:yigFU9vqHzYiE8U
 golang.org/x/crypto v0.0.0-20190510104115-cbcb75029529/go.mod h1:yigFU9vqHzYiE8UmvKecakEJjdnWj3jj499lnFckfCI=
 golang.org/x/crypto v0.0.0-20190605123033-f99c8df09eb5/go.mod h1:yigFU9vqHzYiE8UmvKecakEJjdnWj3jj499lnFckfCI=
 golang.org/x/crypto v0.0.0-20190611184440-5c40567a22f8/go.mod h1:yigFU9vqHzYiE8UmvKecakEJjdnWj3jj499lnFckfCI=
-golang.org/x/crypto v0.0.0-20190820162420-60c769a6c586 h1:7KByu05hhLed2MO29w7p1XfZvZ13m8mub3shuVftRs0=
```

Comment:
Doesn't this remove a layer of assurance? 

It's been a while since I used Go, but IIRC this line is saying that the hash of the entire module zip file is that one and it ensures your entire module zip is correct. 

The lines with /go.mod are only giving the hash of the go.mod file in that module, so you're only ensuring that the dependencies haven't been tampered with instead of the whole module.

So what this change is doing is dropping the tamper protection for only 4 of the deps in this file, which to me seems a little... weird.

Reply:
I mean I could be out of date in my understanding - I guess if you remove these lines it will go to the [Checksum Database](https://go.dev/ref/mod#checksum-database), but it seems weird to do that just for 4/X modules in the go.sum file which is what triggered me.